internal/pkg/cli/env_init.go

```diff
@@ -300,6 +326,93 @@ func (o *initEnvOpts) askCustomizedResources() error {
 	if o.NoCustomResources {
 		return nil
 	}
+	adjustOrImport, err := o.prompt.SelectOne(
```

Comment:
We should try to use the `WithFinalMessage` option while prompting to provide a consistent experience but, in this situation it's a bit tricky since the options are a bit verbose. Just something to keep in mind :)